lib/chat/toolChains/toolChains.ts

```diff
@@ -16,6 +16,8 @@ export type PrepareStepResult = {
 // Map specific tools to their required models
 export const TOOL_MODEL_MAP: Record<string, LanguageModel> = {
   update_account_info: "gemini-2.5-pro",
+  create_knowledge_base: "openai/gpt-5-mini",
+  generate_txt_file: "openai/gpt-5-mini",
```

Comment:
unrelated to this pull request:
why are you using `TOOL_MODEL_MAP` for models rather than following the existing pattern to using the toolChain response?

Reply:
Personally, I think you'll find the `generate_txt_file` tool to be improved if you pull the latest changes I've just shipped to `main` and you can then remove these changes to `lib/chat/toolChains/toolChains.ts`